scripts/create-symlinks.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# Usage in the project root folder:
+# ./scripts/create-symlinks.sh PACKAGE_NAME
+#
+# Usage in package folder
+# ../../scripts/create-symlinks.sh
+
+packageDir=".";
```

Comment:
The root directory is a one-off situation, we shouldn't need to cover that case.

Reply:
This was in case of running script inside the package folder. But i think it's better to run the script from only one place.